components/dashboard/src/menu/Menu.tsx

```diff
@@ -215,15 +223,27 @@ const UserMenu: FC<UserMenuProps> = ({ user, className, withAdminLink, withFeedb
                 href: "https://www.gitpod.io/support/",
                 target: "_blank",
                 rel: "noreferrer",
-                separator: true,
-            },
-            ...extraSection,
-            {
-                title: "Log out",
-                href: gitpodHostUrl.asApiLogout().toString(),
+                separator: !isGitpodIo,
             },
         ];
-    }, [extraSection, user]);
+
+        if (isGitpodIo) {
+            baseEntries.push({
+                title: "Feedback",
+                onClick: onFeedback,
+                separator: true,
+            });
+        }
+
+        baseEntries.push(...extraSection);
```

Comment:
nit: because we're getting rid of the Feedback item here, let's call it `adminSection`

Reply:
Resolved in 657d73f